File: ClientApp/src/app/interfaces/issue.ts
import { User } from "./User";

export class Issue {
  id: string
  title: string = "";
  category: string = "";
  dueDate: string = "2022-09-17T06:54:10.835Z";
  estimate: string = "";
  importance: string = "";
  status: string = "todo"
  user: User;
}

File: dotnet/Models/Issue.cs
namespace IDO.Models
{
  public class Issue
  {
    public int? Id { get; set; }

    public string? Title { get; set; } 
    public string? Category { get; set; } 

    public string? Status { get; set; } 

    public string? Estimate { get; set; } 

    public DateTime? DueDate { get; set; }

    public string? Importance { get; set; }

    public User? User { get; set; }
  }

  public enum Priority
  {
    Low, Medium, High
  }
  public enum Status
  {
    todo, doing, done
  }
}

